CONTRIBUTING.rst

```diff
@@ -0,0 +1,126 @@
+============
+Contributing
+============
+
+First off, thanks for taking the time to contribute!
+
+Contributions are welcome by anybody and everybody. We are not kidding! Looking for help ? Join us on `Slack`_ by requesting an `invite`_.
```

Comment:
Random underscore at the end?

Also probably worth linking to the invite page.